tutorial.md

```diff
@@ -104,7 +104,16 @@ class MyScaffold extends StatelessComponent {
   }
 }
 
-void main() => runApp(new MyScaffold());
+void main() {
+  runApp(
+    new MaterialApp(
+      title: 'My app',
+      routes: {
+        '/': (RouteArguments args) => new MyScaffold()
+      }
+    )
+  );
+}
 ```
 
 The `MyToolBar` component creates a `Container` with a height of 56
```

Comment:
I'll try to add something, but we'll need to clean this section up. This is from a section called "basic widgets", but now we've introduced MaterialApp and routes.